src/main/java/com/urswolfer/gerrit/client/rest/http/changes/ChangesRestClient.java

```diff
@@ -106,7 +106,7 @@ public ChangeApi id(int id) throws RestApiException {
     @Override
     public ChangeApi id(String id) throws RestApiException {
         return new ChangeApiRestClient(gerritRestClient, this, changesParser, commentsParser,
-            fileInfoParser, diffInfoParser, suggestedReviewerInfoParser, id);
+            fileInfoParser, diffInfoParser, suggestedReviewerInfoParser, null, id);
```

Comment:
I think you should pass AccountsParser here - otherwise it is always null and will result in an NullPointerException in ChangeApiRestClient?!